src/Game/Systems/ScoreScreenSystem.cpp

```diff
@@ -0,0 +1,157 @@
+#include "Game/Systems/ScoreScreenSystem.h"
+
+
+ScoreScreenSystem::ScoreScreenSystem(SystemParams params)
+    : System(params)
+    , PureSystem("ScoreScreen")
+{
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerDeath, &ScoreScreenSystem::OnPlayerDeath);
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerSpawned, &ScoreScreenSystem::OnPlayerSpawn);
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerConnected, &ScoreScreenSystem::OnPlayerConnected);
+    EVENT_SUBSCRIBE_MEMBER(m_EPlayerDisconnected, &ScoreScreenSystem::OnPlayerDisconnected);
+}
+
+void ScoreScreenSystem::UpdateComponent(EntityWrapper& entity, ComponentWrapper& capturePoint, double dt)
+{
+    if (!IsServer) {
+        return;
+    }
+
+    if(!entity.HasComponent("ScoreScreen")){
+        return;
+    }
+
+    int redTeamEnum;
+    int blueTeamEnum;
+    int spectatorTeamEnum;
+    int currentTeam = 0;
+
```

Comment:
"int redTeamEnum, int blueTeamEnum, int spectatorTeamEnum, int currentTeam = 0" not used?